src/main/java/uk/gov/hmcts/dts/fact/services/CourtService.java

```diff
@@ -124,6 +173,11 @@ public List<CourtWithDistance> getNearestCourtsByPostcodeAndAreaOfLaw(final Stri
             .orElseThrow(() -> new InvalidPostcodeException(postcode));
     }
 
+    /**
+     * Get nearest courts by postcode, area of law and local authority.
+     *
+     * @return the list of courts
+     */
     public List<CourtWithDistance> getNearestCourtsByPostcodeAndAreaOfLawAndLocalAuthority(final String postcode, final String areaOfLaw, final Boolean includeClosed) {
```

Comment:
I think the params description for this method may have been missed :)

Reply:
added missing bit